docutron/bin/cli.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+const path = require( 'path' );
```

Comment:
Should we add DocBlocks for dependencies ("External dependencies", etc)? For consistency with other "modules".